Mathlib/FieldTheory/Tower.lean

```diff
@@ -66,31 +65,29 @@ theorem lift_rank_mul_lift_rank :
 $\operatorname{rank}_F(A) = \operatorname{rank}_F(K) * \operatorname{rank}_K(A)$.
 
 This is a simpler version of `lift_rank_mul_lift_rank` with `K` and `A` in the same universe. -/
-theorem rank_mul_rank (F : Type u) (K A : Type v) [CommRing F] [Ring K] [AddCommGroup A]
-    [Algebra F K] [Module K A] [Module F A] [IsScalarTower F K A] [StrongRankCondition F]
-    [StrongRankCondition K] [Module.Free F K] [Module.Free K A] :
+theorem rank_mul_rank (A : Type v) [AddCommGroup A]
+    [Module K A] [Module F A] [IsScalarTower F K A] [Module.Free K A] :
     Module.rank F K * Module.rank K A = Module.rank F A := by
   convert lift_rank_mul_lift_rank F K A <;> rw [lift_id]
 #align rank_mul_rank rank_mul_rank
 
 /-- Tower law: if `A` is a `K`-module and `K` is an extension of `F` then
 $\operatorname{rank}_F(A) = \operatorname{rank}_F(K) * \operatorname{rank}_K(A)$. -/
-theorem FiniteDimensional.finrank_mul_finrank' [Module.Finite F K]
-    [Module.Finite K A] : finrank F K * finrank K A = finrank F A := by
+theorem FiniteDimensional.finrank_mul_finrank' : finrank F K * finrank K A = finrank F A := by
```

Comment:
Is `[Module.Finite F K]` still there? If not, maybe remove `finrank_mul_finrank`?